docs/src/main/asciidoc/secret-management.adoc

```diff
@@ -142,37 +61,45 @@ public class SampleApplication implements CommandLineRunner {
 
 === Advanced Usage
 
-==== Special Characters in Property Name
-Key Vault secret name only support characters in `[0-9a-zA-Z-]`. Refs: link:https://docs.microsoft.com/azure/key-vault/general/about-keys-secrets-certificates#vault-name-and-object-name[Vault-name and Object-name]. If your property name contains other characters, you can use these workarounds:
+1. Special Characters in Property Name. Key Vault secret name only support characters in `[0-9a-zA-Z-]`. Refs: link:https://docs.microsoft.com/azure/key-vault/general/about-keys-secrets-certificates#vault-name-and-object-name[Vault-name and Object-name]. If your property name contains other characters, you can use these workarounds:
+-  Use `-` instead of `.` in secret name. `.` is not supported in secret name. If your application have property name which contain `.`, like `spring.datasource.url`, just replace `.` to `-` when save secret in Azure Key Vault. For example: Save `spring-datasource-url` in Azure Key Vault. In your application, you can still use `spring.datasource.url` to retrieve property value. **NOTE**: This method can not satisfy requirement like `spring.datasource-url`. When you save `spring-datasource-url` in Key Vault, only `spring.datasource.url` and `spring-datasource-url` is supported to retrieve property value, `spring.datasource-url` is not supported. To handle this case, please refer to the following option: Use property placeholders.
+-  Use property placeholders. For example: setting this property in your application.properties: `property.with.special.character_=${propertyWithoutSpecialCharacter}`. The application will get  `propertyWithoutSpecialCharacter` key name and assign its value to `property.with.special.character_`.
 
-===== Use `-` Instead of `.` in Secret Name
+2. Case Sensitive. By default, the secret names are case-insensitive. To enable case-sensitive mode, just set the following property: `spring.cloud.azure.keyvault.secret.property-sources[].case-sensitive=true`.
 
-`.` is not supported in secret name. If your application have property name which contain `.`, like `spring.datasource.url`, just replace `.` to `-` when save secret in Azure Key Vault. For example: Save `spring-datasource-url` in Azure Key Vault. In your application, you can still use `spring.datasource.url` to retrieve property value.
+3. Not retrieve all secrets in Key Vault. If you stored 1000 secrets in the Key Vault, and you just want to use 3 of them. You can list the 3 secret names by `spring.cloud.azure.keyvault.secret.property-sources[].secret-keys`.
```

Comment:
suggestion:
something like this format
<img width="412" alt="image" src="https://user-images.githubusercontent.com/4465723/159675784-2b9b0d42-ed3f-47e0-ae5a-33651cc93b75.png">

Reply:
Updated.